INFO:root:Contract //root/smartbugs/dataset/int_bugs_9/0604/repair_tru/INT_repair_mutation_for_inttruncate_151_0.sol:INT:
INFO:symExec:Running, please wait...
INFO:symExec:	============ Results ===========
INFO:symExec:	  EVM code coverage: 	 70.7%
INFO:symExec:	  Arithmetic bugs: 	 True
INFO:symExec:	  └> Overflow bugs: 	 True
//root/smartbugs/dataset/int_bugs_9/0604/repair_tru/INT_repair_mutation_for_inttruncate_151_0.sol:INT:151:1
balanceOf[target] +=uint128(mintedAmount)
^
INFO:symExec:	  └> Underflow bugs: 	 False
INFO:symExec:	  └> Division bugs: 	 False
INFO:symExec:	  └> Modulo bugs:   	 False
INFO:symExec:	  └> Truncation bugs: 	 False
INFO:symExec:	  └> Signedness bugs: 	 False
INFO:symExec:	  Callstack bug: 	 False
INFO:symExec:	  Concurrency bug: 	 False
INFO:symExec:	  Time dependency bug: 	 False
INFO:symExec:	  Reentrancy bug: 	 False
INFO:symExec:	 --- 50.0334479809 seconds ---
INFO:symExec:	====== Analysis Completed ======
INFO:root:Contract //root/smartbugs/dataset/int_bugs_9/0604/repair_tru/INT_repair_mutation_for_inttruncate_151_0.sol:INTToken:
INFO:symExec:Running, please wait...
Traceback (most recent call last):
  File "/root/osiris/taintFlow.py", line 652, in perform_taint_analysis
    propagate_taint(taint, tainted_stack, tainted_memory, tainted_storage, instruction, current_stack, previous_block, current_block, next_blocks, arithmetic_errors, sha3_list, false_positives, strings)
  File "/root/osiris/taintFlow.py", line 418, in propagate_taint
    if arithmetic_error["instruction"].data_in[0] == 32 or arithmetic_error["instruction"].data_in[0] == 31:
  File "/root/osiris/z3/z3.py", line 302, in __nonzero__
    return self.__bool__()
  File "/root/osiris/z3/z3.py", line 310, in __bool__
    return self.arg(0).eq(self.arg(1))
  File "/root/osiris/z3/z3.py", line 922, in arg
    return _to_expr_ref(Z3_get_app_arg(self.ctx_ref(), self.as_ast(), idx), self.ctx)
  File "/root/osiris/z3/z3.py", line 946, in _to_expr_ref
    sk = Z3_get_sort_kind(ctx_ref, Z3_get_sort(ctx_ref, a))
  File "/root/osiris/z3/z3core.py", line 2342, in Z3_get_sort_kind
    _elems.Check(a0)
  File "/root/osiris/z3/z3core.py", line 1326, in Check
    err = self.get_error_code(ctx)
  File "/root/osiris/symExec.py", line 2868, in handler
    raise Exception("timeout")
Exception: timeout
INFO:symExec:	============ Results ===========
INFO:symExec:	  EVM code coverage: 	 69.5%
INFO:symExec:	  Arithmetic bugs: 	 True
INFO:symExec:	  └> Overflow bugs: 	 True
//root/smartbugs/dataset/int_bugs_9/0604/repair_tru/INT_repair_mutation_for_inttruncate_151_0.sol:INTToken:151:1
balanceOf[target] +=uint128(mintedAmount)
^
INFO:symExec:	  └> Underflow bugs: 	 False
INFO:symExec:	  └> Division bugs: 	 False
INFO:symExec:	  └> Modulo bugs:   	 False
INFO:symExec:	  └> Truncation bugs: 	 False
INFO:symExec:	  └> Signedness bugs: 	 False
INFO:symExec:	  Callstack bug: 	 False
INFO:symExec:	  Concurrency bug: 	 False
INFO:symExec:	  Time dependency bug: 	 False
INFO:symExec:	  Reentrancy bug: 	 False
INFO:symExec:	 --- 50.0403170586 seconds ---
INFO:symExec:	====== Analysis Completed ======
INFO:root:Contract //root/smartbugs/dataset/int_bugs_9/0604/repair_tru/INT_repair_mutation_for_inttruncate_151_0.sol:owned:
INFO:symExec:Running, please wait...
INFO:symExec:	============ Results ===========
INFO:symExec:	  EVM code coverage: 	 99.4%
INFO:symExec:	  Arithmetic bugs: 	 False
INFO:symExec:	  └> Overflow bugs: 	 False
INFO:symExec:	  └> Underflow bugs: 	 False
INFO:symExec:	  └> Division bugs: 	 False
INFO:symExec:	  └> Modulo bugs:   	 False
INFO:symExec:	  └> Truncation bugs: 	 False
INFO:symExec:	  └> Signedness bugs: 	 False
INFO:symExec:	  Callstack bug: 	 False
INFO:symExec:	  Concurrency bug: 	 False
INFO:symExec:	  Time dependency bug: 	 False
INFO:symExec:	  Reentrancy bug: 	 False
INFO:symExec:	 --- 0.219037055969 seconds ---
INFO:symExec:	====== Analysis Completed ======
INFO:root:Contract //root/smartbugs/dataset/int_bugs_9/0604/repair_tru/INT_repair_mutation_for_inttruncate_151_0.sol:token:
INFO:symExec:Running, please wait...
INFO:symExec:	============ Results ===========
INFO:symExec:	  EVM code coverage: 	 95.5%
INFO:symExec:	  Arithmetic bugs: 	 False
INFO:symExec:	  └> Overflow bugs: 	 False
INFO:symExec:	  └> Underflow bugs: 	 False
INFO:symExec:	  └> Division bugs: 	 False
INFO:symExec:	  └> Modulo bugs:   	 False
INFO:symExec:	  └> Truncation bugs: 	 False
INFO:symExec:	  └> Signedness bugs: 	 False
INFO:symExec:	  Callstack bug: 	 False
INFO:symExec:	  Concurrency bug: 	 False
INFO:symExec:	  Time dependency bug: 	 False
INFO:symExec:	  Reentrancy bug: 	 False
INFO:symExec:	 --- 50.0650169849 seconds ---
INFO:symExec:	====== Analysis Completed ======

  .oooooo.             o8o            o8o          
 d8P'  `Y8b            `"'            `"'          
888      888  .oooo.o oooo  oooo d8b oooo   .oooo.o
888      888 d88(  "8 `888  `888""8P `888  d88(  "8
888      888 `"Y88b.   888   888      888  `"Y88b. 
`88b    d88' o.  )88b  888   888      888  o.  )88b
 `Y8bood8P'  8""888P' o888o d888b    o888o 8""888P'


!!! SYMBOLIC EXECUTION TIMEOUT !!!

!!! SYMBOLIC EXECUTION TIMEOUT !!!
Unexpected error: <type 'exceptions.Exception'>


!!! SYMBOLIC EXECUTION TIMEOUT !!!